addOns/scripts/src/main/javahelp/org/zaproxy/zap/extension/scripts/resources/help/contents/scanrules.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Script Scan Rules</title>
+</head>
+<body>
+<h1>Script Scan Rules</h1>
```

Comment:
Should we explain when the scripts are evaluated as possible scan rules? It might not be obvious that you need to save the script for the changes to be applied (since we usually use the latest changes).